packages/react/src/FAQ/FAQGroup.tsx

```diff
@@ -60,8 +83,11 @@ function _FAQGroup({children, id, defaultSelectedIndex = 0, ...rest}: FAQGroupPr
           aria-controls={`${instanceId}-panel-${index}`}
           aria-selected={selectedIndex === index}
           onClick={handleTabClick(index)}
+          onKeyDown={handleKeyDown(index)}
           key={index}
           data-testid={`FAQGroup-tab-${index + 1}`}
+          tabIndex={selectedIndex === index ? null : -1}
+          ref={selectedIndex === index ? selectedTabRef : null}
```

Comment:
Just inverting the condition here to put truthy value first...

```suggestion
          tabIndex={selectedIndex !== index ? -1 : undefined}
          ref={selectedIndex === index ? selectedTabRef : undefined}
```

Reply:
Fixed in [5c9b95c](https://github.com/primer/brand/pull/596/commits/5c9b95c462211e48f80224534a07ca0cc5de04c7)